src/charm.py

```diff
@@ -142,6 +164,53 @@ def _run_sql_migration(self, set_timeout: bool) -> None:
         stdout, _ = process.wait_output()
         logger.info(f"Executing automigration: {stdout}")
 
+    def _update_config_restart_service(self, event: HookEvent) -> None:
```

Comment:
This method name is confusing. `update config restart service`? The method handles statuses and (sometimes) pushes a file to the workload.

Reply:
Does `_handle_status_update_config` sound better?